lib/src/components/Tooltip/tooltip.types.ts

```diff
@@ -0,0 +1,31 @@
+import { FC } from "react";
+
+type divType = Omit<JSX.IntrinsicElements["div"], "onChange">;
+
+export type TooltipType = divType & {
+  position?:
+    | "bottom"
+    | "left"
+    | "right"
+    | "top"
+    | "bottom-end"
+    | "bottom-start"
+    | "left-end"
+    | "left-start"
+    | "right-end"
+    | "right-start"
+    | "top-end"
+    | "top-start";
```

Comment:
abstract this to a different types and use it here, It will be easier to migrate in the future